language/regexp/character_classes_spec.rb

```diff
@@ -96,9 +96,11 @@
     "\x7F".match(/[[:ascii:]]/).to_a.should == ["\x7F"]
   end
 
-  it "doesn't match non-ASCII characters with [[:ascii:]]" do
-    ('/[[:ascii:]]/').match("\u{80}").should be_nil
-    ('/[[:ascii:]]/').match("\u{9898}").should be_nil
+  not_compliant_on :opal do
+    it "doesn't match non-ASCII characters with [[:ascii:]]" do
+      ('/[[:ascii:]]/').match("\u{80}").should be_nil
+      ('/[[:ascii:]]/').match("\u{9898}").should be_nil
+    end
   end
```

Comment:
@eregon hehe, it's javascript, *expect the unexpected*(tm) :stuck_out_tongue_winking_eye: Pasting `s = "\u{80}";` into Chrome console:
```
s = "\u{80}";
VM106:2 Uncaught SyntaxError: Unexpected token ILLEGAL
    at Object.InjectedScript._evaluateOn (<anonymous>:895:140)
    at Object.InjectedScript._evaluateAndWrap (<anonymous>:828:34)
    at Object.InjectedScript.evaluate (<anonymous>:694:21)
```